sdk/schemaregistry/schema-registry/README.md

```diff
@@ -63,12 +63,6 @@ const client = new SchemaRegistryClient("<fullyQualifiedNamespace>", new Default
 `SchemaRegistryClient` provides the API for storing and retrieving schemas in
 schema registry.
 
-### SchemaRegistry serializers
-
-- [@azure/schema-registry-avro](https://www.npmjs.com/package/@azure/schema-registry-avro)
-  is a separate package that uses `SchemaRegistryClient` to pair schema ID along
-  with Avro Binary Encoding.
```

Comment:
Based on @swathipil's suggestion. I will add it back after the improved serializer is released.